src/transformStateWithClones.js

```diff
@@ -28,6 +28,9 @@ function transformStateWithClones(state, actions) {
           delete clone[key];
         }
         break;
+
+      default:
+        break;
```

Comment:
I think it's better to write some info about Error, not just break.
for example : 'Error'

Reply:
fixed =)